src/js/helpers/cleanData.js

```diff
@@ -7,7 +8,15 @@ export const cleanContacts = (contacts) => {
   return contacts.edges.map(({node: contact}) => {
     let cleaned = Object.assign(contact.contact);
     if(cleaned.hours && cleaned.hours.edges) {
-      cleaned.hours = cleaned.hours.edges.map(({ node: hours }) => hours);
+      cleaned.hours = cleaned.hours.edges.map(({ node: hours }) => {
+        hours.startTime = moment(hours.startTime, "HH:mm:ss").format('h:mm A');
+        hours.endTime = moment(hours.endTime, "HH:mm:ss").format('h:mm A');
+        let dayMoment = moment(hours.dayOfWeek, 'dddd');
+        hours.dayOfWeek = dayMoment.format('dddd');
+        hours.day = dayMoment.format('E');
```

Comment:
This is really small, but why not get rid of line 14 and just let line 16 be: 
`hours.day = moment(hours.dayOfWeek, 'dddd').format('E');`

That `let` definition in the middle of the rest kinda threw me off and you're already chaining the `.format()` function on all the other lines.